examples/get-server-time.js

```diff
@@ -0,0 +1,16 @@
+// Get the time from the Redis Server.
+
+import { createClient } from 'redis';
+
+async function getServerTime() {
+  const client = createClient();
+  await client.connect();
+
+  const serverTime = await client.time();
+  // 2022-02-25T12:57:40.000Z { microseconds: 351346 }
+  console.log(serverTime);
+
+  await client.quit();
```

Comment:
should we wrap in `try/catch` block?

Reply:
Doesn't add value to the point of the example, so I'd say no.